views/index.erb

```diff
@@ -1,14 +1,35 @@
 <% if @profile %>
-  <h3><%= @profile["first_name"] %>'s Amazing Profile</h3>
+
+  <div id="picNheader">
+    <div id="profilePic" >
+      <img src='https://graph.facebook.com/37109651/picture?width=200&height=200' />
+    </div>
+
+    <h3 id="profileHeader"><%= @profile["first_name"] %>'s Amazing Profile<br><%= @profile["about"] %></h3>
+  </div>
 
   <!-- TODO- Perhaps re-jig this to fit your new design / layout -->
+
   <% @profile.each do |field, value| %>
```

Comment:
Maybe you don't need to iterate over `@profile`? If you indexed into this variable, like you would a regular hash, it would cut down on the need for the `if field == "field name"` logic that is proliferating throughout this code.